File: z-clients/javascript/src/models/transformOut.ts
// this file is @generated

export interface TransformOut {
    /** JSON-encoded value returned by the script's `handler` function. */
    output: string;
}

export const TransformOutSerializer = {
    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _fromJsonObject(object: any): TransformOut {
        return {
            output: object['output'],
        };
    },

    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _toJsonObject(self: TransformOut): any {
        return {
            'output': self.output,
        };
    }
}

File: z-clients/rust/src/models/transform_out.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, Deserialize, Serialize)]
pub struct TransformOut {
    /// JSON-encoded value returned by the script's `handler` function.
    pub output: String,
}

impl TransformOut {
    pub fn new(output: String) -> Self {
        Self { output }
    }
}
